src/Microsoft.AspNetCore.SignalR.Core/Internal/DefaultHubDispatcher.cs

```diff
@@ -225,6 +237,11 @@ public override IReadOnlyList<Type> GetParameterTypes(string methodName)
 
         private string BuildUnexpectedErrorMessage(string methodName, Exception exception)
         {
+            if (!_hubOptions.EnableDetailedErrors)
+            {
+                return "write an error";
```

Comment:
"An unexpected error occurred invoking '{methodName}' on the server."

The method name isn't secret, the client already knew it.